File: angular/src/app/Shared/Models/departure.ts
import { Flight } from "./flight";
import { Crew } from "./crew";
import { Aircraft } from "./aircraft";

export class Departure {
    constructor(
        public id?: number,
        public flight?: Flight,
        public departureDate?: Date,
        public crew?: Crew,
        public aircraft?: Aircraft) { }
}

File: Airport/Shared/DTO/Departure.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Shared.DTO
{
    public class Departure
    {
        public int Id { get; set; }
        [Required]
        public Flight Flight { get; set; }
        [Required]
        public DateTime DepartureDate { get; set; }
        [Required]
        public Crew Crew { get; set; }
        [Required]
        public Aircraft Aircraft { get; set; }
    }
}
